app/src/main/java/org/simple/clinic/newentry/PatientEntryScreenController.kt

```diff
@@ -330,20 +259,13 @@ class PatientEntryScreenController @Inject constructor(
         .ofType<PatientGenderChanged>()
         .map { { ui: Ui -> ui.showMissingGenderError(false) } }
 
-    val noPhoneErrorResets = Observable
-        .merge(events.ofType<PatientPhoneNumberTextChanged>(), events.ofType<PatientNoPhoneNumberToggled>())
-        .map { { ui: Ui -> ui.showEmptyPhoneNumberError(false) } }
-
-    val lessPhoneLengthResets = Observable
-        .merge(events.ofType<PatientPhoneNumberTextChanged>(), events.ofType<PatientNoPhoneNumberToggled>())
+    val lessPhoneLengthResets = events.ofType<PatientPhoneNumberTextChanged>()
```

Comment:
Whoops, hey can you please rename this variable to `phoneLengthTooShortResets`? This rename was supposed to happen in an earlier PR by Sanchita, but I missed that.

Reply:
Same for `morePhoneLengthResets`